index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Skeleton Portfolio</title>
+    <link rel="stylesheet" href="CSS/main.css">
+  </head>
+  <body>
+    <!-- header section! -->
+    <header>
+      <nav>
+        <div class="logo">
+          <a href="#"><h2>My logo</h2></a>
+        </div>
+        <div class="menu"><img src="images/Enabled.png" alt="" /></div>
+      </nav>
+    </header>
+
+    <!-- main section! -->
+    <main>
+      <!-- hello section! -->
+      <section class="hello">
+        <div class="hellosection">
+          <div class="box box1"><h1>I’m Noureddine Glad to see you!</h1></div>
+          <div class="box box2">
+            <p>
+              I’m a software developer! I can help you build a product , feature
+              or website Look through some of my work and experience! If you
+              like what you see and have a project you need coded, don’t
+              hestiate to contact me.
+            </p>
+          </div>
+          <div class="box box3">
+            <div><h3 class="hsocial">LET'S CONNECT</h3></div>
+            <div>
+              <ul>
+                <li class="social">
+                  <a href="#"><img src="images/twitter.svg" alt="" /></a>
+                </li>
+                <li class="social">
+                  <a href="#"><img src="images/Linkedin.svg" alt="" /></a>
+                </li>
+                <li class="social">
+                  <a href="#"><img src="images/M.svg" alt="" /></a>
+                </li>
+                <li class="social">
+                  <a href="#"><img src="images/cat.svg" alt="" /></a>
+                </li>
+                <li class="social">
+                  <a href="#"><img src="images/Y.svg" alt="" /></a>
```

Comment:
- [ ] [OPTIONAL] It will be better if you will give to `alt` a value - for example `alt="twitter"` - do not skip it empty 👍

Reply:
alt filled!

Reply:
- [ ] Please fill all `alt` attributes by the right description to each image as the last reviewer mentioned.